docs/getting-started/android/screenshots.md

```diff
@@ -1,7 +1,123 @@
 # fastlane screenshots for Android
 
-To be written soon, this will contain
+Your app screenshots are probably the most important thing when it comes to convincing potential users to download or purchase your app. Unfortunately, many apps don't do screenshots well. This is often because taking screenshots of your app and then preparing them for display is time consuming, and it's hard to get everything right and consistent! For example:
 
-- Getting started with _screengrab_
-- Uploading the screenshots using _supply_
+- Are the screenshots in the App Store inconsistent with your latest app design?
+- Is your app localized into many languages that require different screenshots for each?
+- Have you made sure that no loading indicators are showing?
```

Comment:
Not exactly, and I removed a bunch of other references to such, so I just missed this one!

Will remove